console/apps/web-ui/public/config.js

```
@@ -18,63 +18,54 @@
-  gatewayControlPlaneUrl: "http://localhost:9243",
-  gatewayVersion: "v0.11.0",
-  ampVersion: "v0.17.0",
-  instrumentationUrl: "http://localhost:22893/otel",
-  agentManagerInternalBaseUrl: "http://host.docker.internal:9000",
-  agentManagerInternalCpHost: "host.docker.internal:9243",
-  guardrailsCatalogUrl:
-    "https://db720294-98fd-40f4-85a1-cc6a3b65bc9a-prod.e1-us-east-azure.choreoapis.dev/api-platform/policy-hub-api/policy-hub-public/v1.0/policies?categories=Guardrails,AI&limit=100",
-  guardrailsDefinitionBaseUrl:
-    "https://db720294-98fd-40f4-85a1-cc6a3b65bc9a-prod.e1-us-east-azure.choreoapis.dev/api-platform/policy-hub-api/policy-hub-public/v1.0/policies",
+  disableAuth: 'false' === 'true',
+  rbacEnabled: 'true' === 'true',
```

Biome (2.5.3)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)